README.md

```diff
@@ -1,37 +1,220 @@
+
+<br />
+
+### **Zambia Invasive Species Tracking**
+This invasive species tracking application was designed to plot the detection of the Fall Army Worm (FAW) across farms in Zambia. The result of which would help governmental agriculture decision makers in focussing remediation efforts of FAW. To achieve this we worked with data aggregated via analog phone text messaging that was collected by an company in Africa called TextIt. We were not able to successfully work with their API but we were able to export data in a format that allowed us to store responses in a relational way to more easily work with this data. This data was gathered every 2 weeks during their grow season and plotted via the text campaign date. More information about this project can be found on our [FAW environment](http://faw.environment.arizona.edu) site.
+
+![Zambia FAW Distribution](https://github.com/uaenvironment/uaenvironment.github.io/blob/master/images/zambia_faw.png?raw=true)
+
+<br />
+
+# Publications
+Guido Z, McMahan B, Hoy D, Larsen C, Delgado B, Granillo RL III and Crimmins MA (2022) Public Engagement on Weather and Climate with a Monsoon Forecasting Game. Bulletin of the American Meteorological Society (IN REVIEW)
+
+Dharma Hoy, Calvin Larsen, Rey Granillo III, UA's Arizona Institute for Resilient Environments and Societies. (2022). uaenvironment/monsoon-game-scoring-method: v1.0.1 (v1.0.1). Zenodo. https://doi.org/10.5281/zenodo.6878318
+
+McMahan B, Granillo RL III, Delgado B, Herrera M and Crimmins MA (2021) [Curating and Visualizing Dense Networks of Monsoon Precipitation Data: Integrating Computer Science Into Forward Looking Climate Services Development](https://www.frontiersin.org/articles/10.3389/fclim.2021.602573/full). Front. Clim. 3:602573. doi: 10.3389/fclim.2021.602573
+
+<br />
+
+# Student Contributors
+
+## Current:
+<a href="https://github.com/rose-prendergast">
+  <img src="https://github.com/rose-prendergast.png" width="25"> Rose Predergast
+</a>
+
+<a href="https://github.com/dharmahoy">
+  <img src="https://github.com/dharmahoy.png" width="25"> Dharma Hoy
+</a>
+
+## Former:
+<a href="https://github.com/francesslater">
+  <img src="https://github.com/francesslater.png" width="25"> Franny Slater
+</a>
+
+<a href="https://github.com/calvinlarsen">
+  <img src="https://github.com/calvinlarsen.png" width="25"> CJ Larsen
+</a>
+
+<a href="https://github.com/gbdelgado">
+  <img src="https://github.com/gbdelgado.png" width="25"> Benni Delgado
+</a>
+
+<a href="https://github.com/mauherrerag">
+  <img src="https://github.com/mauherrerag.png" width="25"> Mau Herrera
+</a>
+
+<a href="https://www.linkedin.com/in/laurenktran">
+  <img src="https://media-exp1.licdn.com/dms/image/C5603AQFKAhA8stmqjA/profile-displayphoto-shrink_200_200/0/1574332029734?e=2147483647&v=beta&t=CV57gJAfRYsQn63pIfOLICmKkyVGngB9zx3Z7Hv5pd8" width="25"> Lauren Tran
+</a>
```

Comment:
Might be a github quirk, but when I select  "Display the rich diff", these all render as one big link.